staging/src/k8s.io/component-helpers/resource/helpers.go

```diff
@@ -46,15 +47,68 @@ type PodResourcesOptions struct {
 	// NonMissingContainerRequests if provided will replace any missing container level requests for the specified resources
 	// with the given values.  If the requests for those resources are explicitly set, even if zero, they will not be modified.
 	NonMissingContainerRequests v1.ResourceList
+	// PodLevelResourcesEnabled indicates that the pod-level resources feature gate
+	// is enabled.
+	PodLevelResourcesEnabled bool
```

Comment:
maybe the name could be more similar to `ExcludeOverhead`: `ExcludePodLevelResources`